CHANGELOG_iOS.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+## [1.75.179](https://github.com/brave/brave-browser/releases/tag/v1.75.179)
+
+### Web3
+
+ - Added new updates to the transaction status screen. ([#26146](https://github.com/brave/brave-core/pull/26146))
+ - Added “Speed up” button on wallet transaction status screen. ([#41800](https://github.com/brave/brave-browser/issues/41800))
+ - Added “Cancel Transaction” button on wallet transaction status screen. ([#41799](https://github.com/brave/brave-browser/issues/41799))
+ - Added support for resolving new Unstoppable Domains TLDs. ([#42368](https://github.com/brave/brave-browser/issues/42368))
+
+### General
+
+ - Added new customizable menu UI behind the “#brave-use-modern-browser-menu” flag. ([#42836](https://github.com/brave/brave-browser/issues/42836))
```

Comment:
We usually don't mention specific flags being enabled for features when using Griffin. Lets use the same approach we do with `Desktop` & `Android`. Example:

```
 - Added new customizable hamburger menu (being rolled out in phases using Griffin starting at 25%). ([#42836](https://github.com/brave/brave-browser/issues/42836))
 ```